src/main/java/uk/gov/hmcts/reform/hmc/service/LinkedHearingGroupServiceImpl.java

```diff
@@ -221,11 +245,33 @@ private void validateUnlinkingHearingsWillNotHaveStartDateInThePast(List<Hearing
         return max.isPresent() ? max.get().getValue() : List.of();
     }
 
-    private void deleteFromLinkedGroupDetails(List<HearingEntity> linkedGroupHearings, Long hearingGroupId) {
+    private void deleteFromLinkedGroupDetails(List<HearingEntity> linkedGroupHearings) {
+        LinkedGroupDetails linkedGroupDetails = linkedGroupHearings.get(0).getLinkedGroupDetails();
+        saveLinkedGroupDetailsAudit(linkedGroupDetails);
         linkedGroupHearings.forEach(hearingEntity -> {
-            // TODO: unlink hearingEntity from the group and persist - https://tools.hmcts.net/jira/browse/HMAN-96
+            saveLinkedHearingDetailsAudit(hearingEntity);
         });
```

Comment:
`  linkedGroupHearings.forEach(hearingEntity -> saveLinkedHearingDetailsAudit(hearingEntity));`

Reply:
updated the code